tool_quality_checks/correct_import_file.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+"""#
+Created on Tue Aug 13 08:14:18 2019
+@author: chris.kerklaan - N&S
+"""
+
+# system imports
+
+# Third-party imports
+import ogr
+import osr
+import logging
+
+# Globals
+DRIVER_OGR_MEM = ogr.GetDriverByName("Memory")
+DRIVER_OGR_SHP = ogr.GetDriverByName("ESRI Shapefile")
+MEM_NUM = 0
+
+# Exceptions
+logger = logging.getLogger(__name__)
+ogr.UseExceptions()
+
+
+def create_mem_ds():
+    """ Creating an ogr datasource in memory"""
+    global MEM_NUM
+    mem_datasource = DRIVER_OGR_MEM.CreateDataSource("mem{}".format(MEM_NUM))
+    MEM_NUM = MEM_NUM + 1
+    return mem_datasource
+
+
+def create_geom_transform(in_spatial_ref, out_epsg):
+    """ Creating a coordinate transformation based on two reference systems """
```

Comment:
De eerste regel is één regel. En normaliter in bevelsvorm ("Doe dit!" "Geef dit terug!")

```suggestion
    """Return coordinate transformation based on two reference systems"""
```